samples/word/20-lists/manage-formatting.yaml

```diff
@@ -0,0 +1,132 @@
+order: 2
+id: word-lists-manage-formatting
+name: List formatting
+description: Shows how to format a list.
+author: OfficeDev
+host: WORD
+api_set:
+    WordApi: '1.3'
+script:
+    content: |
+        $("#insert-list").click(() => tryCatch(insertFormatList));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function insertFormatList() {
+          // Insert a list starting with the third paragraph then set numbering and bullet types of the list items.
+          await Word.run(async (context) => {
+            const paragraphs = context.document.body.paragraphs;
+            paragraphs.load("$none"); // No properties needed.
```

Comment:
Instead of saying "No properties needed.", it might be better to spell out what is being loaded here. Especially since without this, the error you get is "PropertyNotLoaded".